en/firefox/retention/thank-you.ftl

```diff
@@ -0,0 +1,24 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/retention/thank-you/
+
+thank-you-thank-you = Thank You
+thank-you-thank-you-page = Thank you page.
+thank-you-its-all-thanks-to-you = It’s all thanks to you
+thank-you-choosing-firefox-helps = Choosing { -brand-name-firefox } helps { -brand-name-mozilla } make the internet a better place. Here’s what you can do next:
+thank-you-make-firefox-your-default = Make { -brand-name-firefox } your default browser
+thank-you-1-min-action = 1 min action
+thank-you-set-as-your-default = Set as your default
+thank-you-get-firefox-on-your-phone = Get { -brand-name-firefox } on your phone
+thank-you-2-min-install = 2 min install
+thank-you-download-firefox = Download { -brand-name-firefox }
+thank-you-tell-your-friends-about = Tell your friends about { -brand-name-firefox }
+thank-you-1-min-share = 1 min share
+thank-you-join-me-in-the-fight-for = Join me in the fight for an open web by choosing { -brand-name-firefox }!
+thank-you-send-a-tweet = { -brand-name-send } a tweet
```

Comment:
Send is no longer available.  Let's remove it. And let's remove the entries in the  **brands.ftl** file as well